consultation_analyser/consultations/api/views/consultation.py

```diff
@@ -55,3 +60,36 @@ def demographic_options(self, request, pk=None):
         serializer = DemographicOptionSerializer(instance=data, many=True)
 
         return Response(serializer.data)
+
+
+@api_view(["GET"])
```

Comment:
pls can we move these 2 to be DRF actions on the /api/consultation route with detail=False

also lets define some serializers for the request/response payloads

Reply:
also this could do with some tests, probably these should live in `tests/unit/api/views/test_consultation_views.py`